pom.xml

```diff
@@ -121,12 +121,12 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <hk2-bridge.version>2.6.0</hk2-bridge.version>
-        <jersey.version>2.29</jersey.version>
-        <jetty.version>9.4.28.v20200408</jetty.version>
-        <guice.version>4.2.2</guice.version>
-        <slf4j.version>1.7.26</slf4j.version>
-        <jackson.version>2.10.2</jackson.version>
+        <hk2-bridge.version>2.6.1</hk2-bridge.version>
+        <jersey.version>2.36</jersey.version>
+        <jetty.version>10.0.11</jetty.version>
```

Comment:
I did not upgrade to Jetty 11+ since that requires JakartaEE instead of javax. Guice does [not support](https://github.com/google/guice/issues/1383) that yet.